pkg/planner/core/exhaust_physical_plans.go

```diff
@@ -2473,6 +2637,7 @@ func exhaustPhysicalPlans4LogicalJoin(lp base.LogicalPlan, prop *property.Physic
 		}
 		joins = append(joins, mergeJoins...)
 
+		// todo: feel vars.EnhanceIndexJoinBuildV2 and tryToEnumerateIndexJoin(p, prop)
```

Comment:
here is the usage protal, which will be controlled with EnhanceIndexJoinBuildV2 with default ON later